plugins/SliceInfoPlugin/SliceInfo.py

```diff
@@ -119,6 +119,7 @@ def _onWriteStarted(self, output_device):
             }
             for container in global_container_stack.getContainers():
                 container_id = container.getId()
+                container_serialized = ""
```

Comment:
Glad it was cleared up. We've put this on through component test in the meanwhile :smiley: